frontend/pages/hosts/details/cards/Software/DeviceSoftwareTableConfig.tsx

```diff
@@ -76,10 +67,16 @@ export const generateSoftwareTableHeaders = (): ISoftwareTableConfig[] => {
       Header: "Type",
       disableSortBy: true,
       disableGlobalFilter: true,
-      accessor: "source",
-      Cell: (cellProps: ITableStringCellProps) => (
-        <TextCell value={cellProps.cell.value} formatter={formatSoftwareType} />
-      ),
+      id: "source",
+      Cell: (cellProps: ITableStringCellProps) => {
+        const { source, extension_for } = cellProps.row.original;
+        return (
+          <TextCell
+            value={source}
+            formatter={() => formatSoftwareType({ source, extension_for })}
```

Comment:
I think the end result here is the same as just having this `formatSoftwareType` call in the current
scope and plugging its output in as the `value` - seems the `formatter` gets called with the `value`
as an argument, and since this callback takes no argument it's just doing that (calling it at this
scope) anyway.

Side note, this whole `formatter` pattern is suspect, will bring it to frontend sync this week